storagenode/monitor/monitor.go

```diff
@@ -197,5 +197,11 @@ func (service *Service) AvailableBandwidth(ctx context.Context) (_ int64, err er
 		return 0, Error.Wrap(err)
 	}
 	allocatedBandwidth := service.allocatedBandwidth
-	return allocatedBandwidth - usage, nil
+	availableBandwidth := allocatedBandwidth - usage
+
+	mon.IntVal("allocated_bandwidth").Observe(allocatedBandwidth) //locked
+	mon.IntVal("usage_bandwidth").Observe(usage)                  //locked
+	mon.IntVal("available_bandwidth").Observe(availableBandwidth) //locked
```

Comment:
Since any one of these can be derived with the other two, it should only be necessary to have two of them in queries. @benjaminsirb has told me in the past that the datascience team can just do calculations in the query if they need a piece of information that can be derived from others.

Reply:
I would recommend reporting `allocatedBandwidth` and `availableBandwidth` but getting rid of `usage`

Reply:
@pgerbes1 @benjaminsirb Can I keep the moby suggestion or do you prefer keeping other metrics?